src/codeCache.h

```diff
@@ -116,6 +122,15 @@ class CodeCache {
 
     void expand();
     void makeImportsPatchable();
+    void saveImport(ImportId id, void** entry);
+
+    inline void** getImport(ImportId id, ImportType type) {
```

Comment:
I'm not a fan of adding `inline` everywhere, it's just noise.
But with two-dimensional `_imports`, this function is likely not needed at all.